docs/src/content/docs/guides/components.mdx

```diff
@@ -419,19 +419,22 @@ import IconsList from '~/components/icons-list.astro';
 
 Starlight provides a set of common icons that you can display in your content using the `<Icon>` component.
 
-Each `<Icon>` requires a [`name`](#all-icons) and can optionally include a `label`, `size`, and `color` attribute.
+Each `<Icon>` requires a [`name`](#all-icons) and can optionally include a `label` to provide context for screen readers.
+The `size` and `color` attributes can be used to adjust the icon’s appearance using CSS units and color values.
 
 ```mdx
 # src/content/docs/example.mdx
 
 import { Icon } from '@astrojs/starlight/components';
 
 <Icon name="star" color="goldenrod" size="2rem" />
+<Icon name="rocket" color="var(--sl-color-accent)" />
 ```
 
 The code above generates the following on the page:
 
 <Icon name="star" color="goldenrod" size="2rem" />
+<Icon name="rocket" color="var(--sl-color-accent)" />
```

Comment:
Tiny nit, but in dark mode the blue accent has slightly low contrast so I might prefer the text accent colour:

````suggestion
<Icon name="rocket" color="var(--sl-color-text-accent)" />
```

The code above generates the following on the page:

<Icon name="star" color="goldenrod" size="2rem" />
<Icon name="rocket" color="var(--sl-color-text-accent)" />
````